remoteworkitem/github.yaml

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
Wouldn't it make sense to store the recordings in the same folder as the other ones?`
https://github.com/almighty/almighty-core/tree/master/test/data